wiki/osu!stream/de.md

```diff
@@ -0,0 +1,375 @@
+- Der Start, das Ende und jeder Slidertick eines Sliders.
+- Einen Spinner vervollständigen.
+
+Das Folgende setzt den Punktemultiplikator auf null zurück:
+
+- Einen Hit-Circle verfehlen.
+- Den Hold-Circle loszulassen, bevor dieser explodiert ist.
+- Die verbundenen Beats nicht gleichzeitig treffen. (Einen treffen und den anderen verfehlen)
+- Einen Slider zu früh klicken.
+- Den Sliderstart oder einen Slidertick verfehlen.
+- Die Spinnermessleiste nicht vollständig füllen.
+
+Das Folgende wird den Punktemultiplikator weder erhöhen noch zurücksetzen:
+
+- Ein Sliderende verfehlen. (Es wird ein Good! oder ein 100/50 Treffer vergeben)
+- Punktebonus eines Spinners
+
+### Benotung
+
+![](img/Os-Grade.jpg "Ergebnisbildschirm")
+
+**Normale Noten**
+
+- SS = 100% Genauigkeit
+- S = Über 90% Genauigkeit
+- A = Über 80% Genauigkeit
+- B = Über 70% Genauigkeit
+- C = Über 60% Genauigkeit
+- D = Alles andere.
+
+### Lebensanzeige
+
+Nicht wie in osu!, ist die Lebensanzeige anders in osu!stream, da sie auf dem Spielmodus basiert.
+
+Im **leichten Modus ist die Lebensleiste nicht vorhanden**.
+
+![](img/Os-SUN.jpg "Beispiel eines Stream Up! von Normal aus")
+
+Im **Streammodus** erhält der Spieler drei Lebensleisten und startet mit der zweiten zur Hälfte gefüllten Lebensleiste. Spieler bewegen sich zwischen diesen beiden Lebensleisten via den **Stream Up!** und **Stream Down!** Systemen, die aktiviert werden, wenn die Lebensleiste komplett gefüllt oder geleert wird. Nach einer Ankündigung wechselt das Spiel zum neuen "Stream" durch das Austauschen des Hintergrunds und einem nahtlosem Wechsel zu einem höheren oder niedrigeren Schwierigkeitsgrad der Beatmap sowie das Tauschen der Lebensleisten. Wenn ein Stream Down! erscheint bekommt der Spieler temporäre Unbesiegbarkeit, während die Lebensleiste zu einem niedrigeren Schwierigkeitsgrad wechselt.
+
+Im **Expertenmodus wird stattdessen eine volle Lebensleiste vergeben**.
+
+Das Folgende füllt die Lebensleiste:
+
+- Einen Spinner und Hold-Bubbles vervollständigen.
+- Ein osu!, gold good!, green good!, 300er, oder 100er treffen,
+  - Bei Hit-Circles füllen bessere Bewertungen die Lebensleiste mehr. Der letzte Hit-Circle in einer farbigen Combo vergibt mehr Lebenspunkte.
+  - Bei Slidern wird die Lebensleiste abhängig von den vervollständigten Teilen gefüllt, nachdem der Slider beendet wurde.
+
+Das Folgende reduziert die Lebensleiste:
+
+- Einen 50er erhalten.
+- Ein Hit-Object oder Slider verfehlen.
+- Nicht in einem Follow-Circle eines Sliders bleiben.
+- Nicht die Hold-Bubble halten, bis sie explodiert.
+- Einen Spinner nicht drehen.
+- Einen Spinner nicht mit einer "Clear" Nachricht vervollständigen.
+
+Im Gegensatz zu anderen Versionen von osu! reduziert sich die Lebensleiste hier nicht von selbst.
+
+## Fragen
+
+### Warum ist (füge hier irgendeinen urheberrechtlich geschützten Song ein) nicht enthalten?
+
+Wegen der Schwierigkeit eine Lizenz für den oben genannten Song zu bekommen.
```

Comment:
```suggestion
Wegen der Schwierigkeit, eine Lizenz für den oben genannten Song zu bekommen.
```
I don't know when optional commas are correct. Shouldn't be wrong to put it here.